config/environments/test.rb

```diff
@@ -47,4 +47,11 @@
 
   #enables a few aliases - context, should, and should_eventually methods
   config.minitest_spec_rails.mini_shoulda = true
+
+  # High logger level results in less IOps & faster tests
+  config.log_level = :warn
+  I18n.enforce_available_locales = false
```

Comment:
We can't set this unilaterally as our RPMs don't have it, see http://projects.theforeman.org/issues/4274.  Perhaps check if it's available, or handle this separately under that ticket.